src/components/tabController/TabPage.tsx

```diff
@@ -49,9 +51,14 @@ export default function TabPage({
   useAnimatedReaction(() => {
     return targetPage.value === index;
   },
-  isActive => {
+  (isActive, wasActive) => {
     if (isActive) {
       runOnJS(lazyLoad)();
+      runOnJS(setFocused)(true);
```

Comment:
Do we want to make sure this runs only once `lazyLoad` has finished?

Reply:
Don't think it's necessary 
They don't have to be coupled